### PR TITLE
 MEGA2560 the first PPM pulse in the frame longer. #2938 is missing i…

### PR DESCRIPTION
**Pull request by Ingwie**
…n next

### CODE REVIEW

**Comment by Ingwie** [comment]
#3536 is missing in next

**Comment by projectkk2glider** [comment]
Actually this is a port of #2981 into `next` branch.

File: radio/src/pulses/pulses_avr.cpp [hunk 1]
```diff
@@ -94,135 +93,140 @@ uint8_t *pulses2MHzWPtr = pulses2MHz;
...
-  
   if (dt > g_tmr1Latency_max) g_tmr1Latency_max = dt;
   if (dt < g_tmr1Latency_min) g_tmr1Latency_min = dt;
 }
 
 void setupPulsesPPM(uint8_t proto)
 {
-  // Total frame length is a fixed 22.5msec (more than 9 channels is non-standard and requires this to be extended.)
-  // Each channel's pulse is 0.7 to 1.7ms long, with a 0.3ms stop tail, making each compelte cycle 1 to 2ms.
+    // Total frame length is a fixed 22.5msec (more than 9 channels is non-standard and requires this to be extended.)
```
Comment (projectkk2glider) [inline radio/src/pulses/pulses_avr.cpp]: Remove two spaces indentation on this and all following lines that have the same indentation added.

File: radio/src/pulses/pulses_avr.cpp [hunk 2]
```diff
@@ -94,135 +93,140 @@ uint8_t *pulses2MHzWPtr = pulses2MHz;
...
-  0x39c3,0x284a,0x1ad1,0x0b58,0x7fe7,0x6e6e,0x5cf5,0x4d7c,
-  0xc60c,0xd785,0xe51e,0xf497,0x8028,0x91a1,0xa33a,0xb2b3,
-  0x4a44,0x5bcd,0x6956,0x78df,0x0c60,0x1de9,0x2f72,0x3efb,
-  0xd68d,0xc704,0xf59f,0xe416,0x90a9,0x8120,0xb3bb,0xa232,
-  0x5ac5,0x4b4c,0x79d7,0x685e,0x1ce1,0x0d68,0x3ff3,0x2e7a,
-  0xe70e,0xf687,0xc41c,0xd595,0xa12a,0xb0a3,0x8238,0x93b1,
-  0x6b46,0x7acf,0x4854,0x59dd,0x2d62,0x3ceb,0x0e70,0x1ff9,
-  0xf78f,0xe606,0xd49d,0xc514,0xb1ab,0xa022,0x92b9,0x8330,
-  0x7bc7,0x6a4e,0x58d5,0x495c,0x3de3,0x2c6a,0x1ef1,0x0f78
+    0x0000,0x1189,0x2312,0x329b,0x4624,0x57ad,0x6536,0x74bf,
```
Comment (projectkk2glider) [inline radio/src/pulses/pulses_avr.cpp]: Remove two spaces indentation on this and all following lines that have the same indentation added.

**Comment by projectkk2glider** [comment]
Uff, I just realized I am reviewing an already merged pull request :blush: I will make cosmetic changes.

**Comment by bsongis** [comment]
I compared with 'master' and didn't notice any difference. Perhaps 'master' is wrong too ...

**Comment by Ingwie** [comment]
Master was OK and Next is OK too since this pull

**Comment by bsongis** [comment]
Obviously the coding style is not as @projectkk2glider noticed

**Comment by bsongis** [comment]
Not an issue, don't worry.

**Comment by Ingwie** [comment]
I have just copy and paste this (from master to next) :
[Mega2560]
    Fixed: First PPM pulse in the frame was longer (#2981)

**Comment by projectkk2glider** [comment]
@Ingwie don't worry, the original change from `master` had some cosmetic deficiencies which were not discovered until now. I fixed them in the `next` branch and we will leave them in `master`.

**Comment by Ingwie** [comment]
Thank you ! Next is working fine on my mega now ..  Rest the voice prompts to set correctly for AVR.

**Pull request merged**